cypress/features/regression/experimental/decimalInput.feature

```diff
@@ -1,116 +1,103 @@
-    Given I set label to "label"
-      And I check labelInline checkbox
-    When I set inputWidth slider to <width>
+    When I open default "Experimental-Decimal-Input" component in noIFrame with "decimal" json from "experimental" using "<nameOfObject>" object name
     Then inputWidth on preview is <width>
     Examples:
-      | width |
-      | 1     |
-      | 10    |
-      | 100   |
+      | width | nameOfObject  |
+      | 1     | inputWidth1   |
+      | 10    | inputWidth10  |
+      | 100   | inputWidth100 |
 
   @positive
   Scenario Outline: Change Decimal component label align to <labelAlign>
-    Given I set label to "label"
-      And I check labelInline checkbox
-    When I select labelAlign to "<labelAlign>"
+    When I open default "Experimental-Decimal-Input" component in noIFrame with "decimal" json from "experimental" using "<nameOfObject>" object name
     Then label align on preview is set to "<labelAlign>"
     Examples:
-      | labelAlign |
-      | left       |
-      | right      |
-
-  @positive
-  Scenario Outline: Check Decimal component input field will not accept characters except numbers to <label>
-    When I set Decimal input to <label>
-    Then Decimal label is not set to <label>
-    Examples:
-      | label                        |
-      | mp150ú¿¡üßä                  |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | labelAlign | nameOfObject    |
+      | left       | labelAlignLeft  |
+      | right      | labelAlignRight |
 
   @positive
   Scenario: Disable Decimal component
-    When I disable Decimal component
+    When I open default "Experimental-Decimal-Input" component in noIFrame with "decimal" json from "experimental" using "disabled" object name
     Then Decimal component is disabled
 
   @positive
   Scenario: Disable and enable Decimal component
-    When I disable Decimal component
-      And I enable Decimal component
+    When I open default "Experimental-Decimal-Input" component in noIFrame with "decimal" json from "experimental" using "disabledFalse" object name
     Then Decimal component is enabled
 
   @positive
   Scenario: Decimal component is readOnly
-    When I check readOnly checkbox
+    When I open default "Experimental-Decimal-Input" component in noIFrame with "decimal" json from "experimental" using "readOnly" object name
     Then Decimal component is readOnly
 
   @positive
   Scenario: Decimal component is not readOnly
-    When I check readOnly checkbox
-      And I uncheck readOnly checkbox
-    Then Decimal component is not readOnly
\ No newline at end of file
+    When I open default "Experimental-Decimal-Input" component in noIFrame with "decimal" json from "experimental" using "readOnlyFalse" object name
+    Then Decimal component is not readOnly
+
+  @positive
+  Scenario Outline: Check Decimal component input field will not accept characters except numbers to <label>
+    Given I open "Experimental-Decimal-Input" component page in noIFrame
```

Comment:
why you didn't use a new approach here ?

Reply:
That is because we don't need to pass any `parameters`. And `I open "Experimental-Decimal-Input" component page in noIFrame` will open `component` without `iFrame`